src/features/moreTab/more/MoreScreen.tsx

```diff
@@ -288,6 +292,11 @@ const MoreScreen = () => {
         disabled: app.isDeployModeEnabled,
       },
     ]
+
+    if (!SUPPORTED_CURRENCIES[currencyType]) {
```

Comment:
I dont believe this should be needed if we are setting `defaultCurrencyType` to USD

Reply:
I found out that 'CNY' failed.